packages/support/resources/views/components/fieldset.blade.php

```diff
@@ -5,12 +5,12 @@
 <fieldset
     {{
         $attributes->class([
-            'fi-fieldset rounded-xl bg-white p-6 shadow-sm ring-1 ring-gray-950/5 dark:bg-white/5 dark:ring-white/10',
+            'fi-fieldset rounded-xl p-6 shadow-sm border border-gray-200 dark:border-white/10',
         ])
     }}
 >
     @if (filled($label))
-        <legend class="text-sm font-medium leading-6">
+        <legend class="text-sm font-medium leading-6 px-1 -mx-1">
```

Comment:
These changes are no longer needed, are they?

Reply:
We used `-ml-2 px-2` in v2 so restored that

Reply:
It adds a little extra spacing around the legend

Reply:
Okay I'll pull in the changes and properly review later.

Reply:
Anything I can do for this?